koku/masu/database/trino_sql/aws/openshift/managed_aws_openshift_daily.sql

```diff
@@ -105,6 +106,8 @@ cte_array_agg_volumes AS (
     SELECT DISTINCT persistentvolume, csi_volume_handle
     FROM hive.{{schema | sqlsafe}}.openshift_storage_usage_line_items_daily
     WHERE source = {{ocp_source_uuid}}
+        AND persistentvolume != ''
+        AND csi_volume_handle != ''
```

Comment:
If customers have not updated their operator they will not have the csi_volume_handle. So, we need to account for the case where a persistentvolume has been populated, but the `csi_volume_handle` has not.

Reply:
I wondered if that might be a case!! Thanks. I originally just had the check on `persistentvolume` but wasnt sure if that would be enough. Based on what your saying I think that would be the better approach here.